src/client/src/exporters/db.rs

```diff
@@ -163,88 +74,89 @@ impl AuroraClient {
         run_name: &str,
         run_id: &str,
         pipeline_name: &str,
-        data: impl IntoIterator<Item = &Event>,
+        logs: impl IntoIterator<Item = Event>,
     ) -> Result<()> {
-        let now = Instant::now();
+        let now = std::time::Instant::now();
 
         const QUERY: &str = "INSERT INTO batch_jobs_logs (
-            data, job_id, run_name, run_id, pipeline_name, nextflow_session_uuid, job_ids,
-            tags, event_timestamp, ec2_cost_per_hour, cpu_usage, mem_used, processed_dataset,
-            process_status
-            )"; // when updating query, also update params
-        const PARAMS: usize = 13;
+                timestamp, body, severity_text, severity_number,
+                trace_id, span_id,
+                source_type, instrumentation_version, instrumentation_type,
+                environment, pipeline_type, user_operator, organization_id, department,
+                run_id, run_name, pipeline_name,
+                job_id, parent_job_id, child_job_ids, workflow_engine,
+                ec2_cost_per_hour, cpu_usage, mem_used, processed_dataset,
+                process_status,
+                attributes, resource_attributes, tags
+            )";
+        // when updating query, also update params
+        const PARAMS: usize = 29;
 
         fn _push_tuple(mut b: Separated<Postgres, &str>, event: EventInsert) {
-            b.push_bind(event.json_data)
-                .push_bind(event.job_id)
-                .push_bind(event.run_name)
+            b.push_bind(event.event_timestamp.naive_utc())
```

Comment:
@blaginin @Iamdavidonuh just curious, is there in rust a way to push these binds in an more explicative way?
for example `b.push_bind("field_name", event.field_value)`